html/user/per_app_list.php

```diff
@@ -63,13 +67,20 @@ function show_header($is_team, $apps, $appid, $is_total) {
 // show a user or team, with their credit for each app
 //
 function show_row($item, $apps, $is_team, $i) {
```

Comment:
The reason first level caching was mentioned on the title of the pull request is because only the choice of items and the chosen app credit for each of them is cached. A DB lookup still has to be done for the credit of the other apps' credit and caching that too would be a possible future improvement.